<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chefWindow</class>
 <widget class="QMainWindow" name="chefWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="box1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTableWidget" name="table1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>181</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>261</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTableWidget" name="table2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>61</y>
       <width>221</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选择操作</string>
    </property>
    <addaction name="action_R"/>
    <addaction name="action_U"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_R"/>
   <addaction name="action_U"/>
  </widget>
  <action name="action_R">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/buttons/refresh-1.png</normaloff>:/buttons/refresh-1.png</iconset>
   </property>
   <property name="text">
    <string>刷新(&amp;R)</string>
   </property>
  </action>
  <action name="action_U">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/buttons/up-arrow-2.png</normaloff>:/buttons/up-arrow-2.png</iconset>
   </property>
   <property name="text">
    <string>上传绩效(&amp;U)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
